suggestion-bot/src/commands/help.ts

```diff
@@ -2,13 +2,13 @@ import Discord from "discord.js";
 
 module.exports = (msg: Discord.Message) => {
   let reply = "```\n";
-  reply += "!suggest <your suggestion here>\n";
+  reply += ">suggest <your suggestion here>\n";
   reply += "  - Adds your suggestion in databse\n";
-  reply += "!list\n";
+  reply += ">list\n";
   reply += "  - Lists all CURRENT suggestions\n";
-  reply += "!list mine\n";
+  reply += ">list mine\n";
   reply += "  - Lists all YOUR suggestions\n";
-  reply += "!vote <number>\n";
+  reply += ">vote <number>\n";
   reply += "  - Vote a suggestion (use after !list)\n";

```

Comment:
change `!list` to `>list` also

Reply:
also didnt bother to change this cause i will move prefix to the database so it could change per channel or server. but will merge your changes

Reply:
pushed the change in line 12